apps/storybook/src/forms/Field.stories.tsx

```diff
@@ -61,3 +63,11 @@ export const DescriptionAndError: Story = {
     label: "Form label",
   },
 };
+
+export const Info: Story = {
+  args: {
+    children: <></>,
```

Comment:
let's add an `<Input />` here
```suggestion
    children: <Input />,
```